docs/data-sources/users.md

```diff
@@ -25,48 +25,56 @@ data "octopusdeploy_users" "example" {
 
 ### Optional
 
-- `filter` (String) A filter with which to search.
+- `filter` (String) A filter search by username, display name or email
 - `ids` (List of String) A filter to search by a list of IDs.
 - `skip` (Number) A filter to specify the number of items to skip in the response.
-- `space_id` (String) A Space ID to filter by. Will revert what is specified on the provider if not set.
+- `space_id` (String) The space ID associated with this user.
```

Comment:
I think `space_id` should be removed entirely since users aren't space-scoped but I've left it in for backwards compatibility.

Reply:
hmm, good point. Lets mark it as deprecated.